server/spinwick.go

```diff
@@ -523,20 +538,8 @@ func (s *Server) initializeMattermostTestServer(mmURL string, prNumber int) erro
 	config.TeamSettings.ExperimentalViewArchivedChannels = NewBool(true)
 	config.PluginSettings.EnableUploads = NewBool(true)
 	config.ServiceSettings.EnableTesting = NewBool(true)
-	// removed in 5.16
-	//config.ServiceSettings.ExperimentalLdapGroupSync = NewBool(true)
 	config.ServiceSettings.EnableDeveloper = NewBool(true)
 	config.LogSettings.FileLevel = NewString("INFO")
-	config.EmailSettings.FeedbackName = NewString("SpinWick Feedback")
```

Comment:
In the future, it would be nice not to have to set any config manually. Maybe we could create a new group with all matterwick config set in it if the current group doesn't do so already?

Reply:
i will create a group and then remove the things from here! :D

Reply:
group created `qyr58czntbyf3mitxy9gtutckr`

@gabrieljackson PTAL

Reply:
I reviewed the group config. Looks good to me!